deepchecks/utils/features.py

```diff
@@ -163,20 +163,24 @@ def calculate_feature_importance(
                 permutation_failure = f'{e.message}\n using model\'s built-in feature importance instead'
 
     # If there was no force permutation, or it failed tries to take importance from the model
-    if importance is None:
+    # We don't use a built importance in pipelines because the order might have changed
```

Comment:
```suggestion
    # We don't take built-in importance in pipelines because the pipeline is changing the features (for example one-hot encoding) which leads to the inner model features being different than the original dataset features
```